patches/jest-expo+45.0.1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/jest-expo/src/preset/assetFileTransformer.js b/node_modules/jest-expo/src/preset/assetFileTransformer.js
+index c4b0952..dcda667 100644
+--- a/node_modules/jest-expo/src/preset/assetFileTransformer.js
++++ b/node_modules/jest-expo/src/preset/assetFileTransformer.js
+@@ -1,6 +1,6 @@
+ const createCacheKeyFunction = require('@jest/create-cache-key-function').default;
+ 
+ module.exports = {
+-  process: (_, filename) => `module.exports = 1;`,
++  process: (_, filename) => ({code: `module.exports = 1;`}),
```

Comment:
i apply this patch with `patch-package`. Not sure how to install a commit/version of a subpackage though..